apps/notifications/src/app/note-panel/actions.tsx

```diff
@@ -1,26 +1,49 @@
 import { MenuGroup, MenuItem, DropdownMenu } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import { moreVertical } from '@wordpress/icons';
-import { useDispatch } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
 import actions from '../../panel/state/actions';
+import getIsShortcutsPopoverOpen from '../../panel/state/selectors/get-is-shortcuts-popover-open';
+import NoteShortcuts from '../note-shortcuts';
 
 export default function NotePanelActions() {
 	const dispatch = useDispatch();
 
+	const isShortcutsPopoverOpen = useSelector( getIsShortcutsPopoverOpen );
```

Comment:
Not a blocker: I'm not sure whether we still need global state for it. Maybe we can simply keep the `isShortcutsPopoverOpen` in this component 🤔